cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/distributed/AbstractProgramTwillRunnable.java

```diff
@@ -231,6 +235,10 @@ public void stop() {
     }
   }
 
+  protected Arguments resolveScope(ProgramOptions programOptions) {
```

Comment:
protected method need to have javadoc. Need to specify what are the expectations if sub-class overrides it.